pyprof/nvtx/nvmarker.py

```diff
@@ -183,6 +183,23 @@ def get_trace_info(op_name):
             #
             fn_name = frame.name
 
+            # Capture class name for any special class functions
+            #
+            if (frame.name.startswith("__")):
+                # ins.stack order is backwards with respect to traceback, so index from
+                # the back instead of the front
+                #
+                index = -(i+1)
+                detailed_frame = ins.stack()[index][0]
```

Comment:
The inspect library was causing memory issues previously.  What guarantee do we have that this change does not reintroduce the GPU OOM issue we saw previously?